.circleci/real_config.yml

```
@@ -2731,8 +2750,7 @@ workflows:
       - test-cli:
           matrix:
             parameters:
-              executor-name:
-                ["python-38", "python-39", "win/default"]
+              executor-name: ["python-38", "python-39", "win/default"]
```

Why is this an inline array?  If cleaning it up, just make it "real" yaml. :)
```suggestion
              executor-name:
                - "python-38"
                - "python-39"
                - "win/default"
```

---

that was my autoformatter

---

I've come inches from fixing that a couple of times, but it's always out of scope.  I guess either put it back down there (since it's out of scope here too) or fix it "right-er."